src/documentation/common/CodeEmbed.tsx

```diff
@@ -89,16 +89,21 @@ const CodeEmbed = ({
       `${toolkitType}-${parentSlug}`
     );
   }, [actions, codeWithImports, parentSlug, toolkitType]);
+  const logging = useLogging();
   const projectActions = useProjectActions();
   const handleOpenIdea = useCallback(async () => {
+    logging.event({
+      type: "idea-opened",
```

Comment:
```suggestion
      type: "idea-open",
```

Tweak for tense consistency.